src/brogue/Monsters.c

```diff
@@ -687,7 +687,11 @@ boolean spawnMinions(short hordeID, creature *leader, boolean summoned) {
         }
 
         for (iMember = 0; iMember < count; iMember++) {
-            monst = generateMonster(theHorde->memberType[iSpecies], true, !summoned);
+            if (rogue.patchVersion >=4) {
```

Comment:
why do we need the version check? is it to ensure saved replays compatibility with previous version?

Reply:
Yep. Earlier recordings will go out of sync otherwise.

Reply:
do we only guarantee replays compatibility within the same minor version (hence only need to check patchVersion)? I guess so - otherwise I would expect seeing similar version checks all over the codebase here :)

Reply:
Yeah patch version checking was first introduced with 1.9.x. From what I understand, as part of the 1.10 release, all the version checking code will be removed.